docs/docs/guides/otel.md

```diff
@@ -160,3 +160,34 @@ kubectl port-forward deployment/metrics-operator 9999 -n keptn-system
 ```
 
 You can access the metrics from your browser at: `http://localhost:9999`
+
+# Advanced tracing configurations in Keptn: Linking traces
+
+In Keptn you can connect multiple traces, for instance to connect deployments 
+of the same application through different stages. 
+To create connections between the traces of versions of your application, you can enrich the
+`KeptnAppContext` resource with
+[Opentelemetry span links](https://opentelemetry.io/docs/concepts/signals/traces/#span-links).
+You can retrieve the span link from the JSON representation of the trace in Jaeger, where
+it has the following structure:
+
+```yaml
+00-<trace-id>-<span-id>-01
+```
+
+This span link needs to be added the `KeptnAppContext` resource, 
+which ensures the connection
+between traces of different versions of the application.
```

Comment:
```suggestion
Use this value to populate the `spanLinks` field
of your `KeptnAppContext` resource
to connect traces of different versions of the application.
```
This makes it so that I find this info if I do a ctrl-f for `spanLinks`